apps/web-giddh/src/app/purchase/purchase-order-preview/purchase-order-preview.component.html

```diff
@@ -0,0 +1,503 @@
+                                            </span>
+                                        </div>
+
+                                        <!-- <div class="history-btn">
+                                            <button class="btn btn-primary" (click)="toggleActivityHistoryAsidePane()">
+                                                <i class="icon-history-icon font-12"></i> History</button>
+                                        </div> -->
+
+                                    </div>
+                                </div>
+
+                            </div>
+                        </div>
+                    </div>
+                    <div class="invoice-image-section">
+
+                        <div class="relative">
+                            <div class="loader"></div>
+                        </div>
+
+                        <!-- <div class="error" hidden>
+                            <span>
+                                Invoice Preview is not available right now!Please try again..
+                            </span>
+                        </div>
+                        <div class="preview-not-supported-attachment" hidden>
+                            <span class="text-warning">Supported format for preview: Images (.jpg, .jpeg, .png) and PDF
+                                (.pdf); to print other formats, download them. </span>
+                        </div>
+                        <div class="upload-excel center-block box mrT2 dashed-border">
+                            <input type="file" id="csv-upload-input" placeholder="Upload an EXCEL or CSV file"
+                                droppable="true" class="form-control" />
+                            <div class="clearfix pos-rel width100">
+                                <div class="file-input-wrapper text-center">
+                                    <div class="clearfix mrB2">
+                                        <img src="assets/images/new/import_icon.svg" />
+                                    </div>
+                                    <div class="clearfix pr-2 pd-2">
+                                        <label class="clearfix text-light-2">Drop file to upload</label>
+                                        <div class="mrT3">
+                                            <label class="file-uplod-penal" for="csv-upload-input"> <span>Browse
+                                                    file</span></label>
+                                        </div>
+                                    </div>
+                                </div>
+                            </div>
+                        </div>
+                        <div class="fixed-ng2pdfview pdf-preview" hidden>
+                            <img class="image-preview" />
+                        </div> -->
+                        <!-- template -->
+
+                        <body id="default" *ngIf="false">
```

Comment:
Please check this block will be absent on UI (it also contains static text)

Reply:
commented